### PR TITLE
Make the Guides front page more appealing

### PR DESCRIPTION
**Pull request by cvphelps**
# Description

- **Problem:** The front page of `guides` is a wall of long sentences.
- **Solution:** Add a video, diagram, and short scannable links for easy glancing.

## Before
image

## After
image

# Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [X] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ ] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [ ] I merged the latest changes from `main` into my feature branch before submitting this PR.

## Weird errors when building Japanese docs locally
image

image

### CODE REVIEW

**Comment by cvphelps** [comment]
@ngrayluna how does this look to you? I'm a little confused about the proper format for links. Some of them worked when I did /intro.md but others seemed to want me to skip `.md` endings. For example:

image

**Comment by ngrayluna** [comment]
Yes. Confusing indeed. 

Per recommendation of our Eng and the [official Docusaurus docs](https://docusaurus.io/docs/markdown-features/links), we use relative files paths [point to specific file and include the  file extension (`*.md`)].   

My commit above fixes the links.

File: docs/guides/intro.md [hunk 1]
```diff
@@ -1,36 +1,39 @@
...
-1. Explore our [Integrations guide](./integrations/intro.md) and our [W&B Easy Integration YouTube](https://www.youtube.com/playlist?list=PLD80i8An1OEGDADxOBaH71ZwieZ9nmPGC) playlist for information on how to integrate W&B with your preferred machine learning framework.
-1. View the [API Reference guide](../ref/README.md) for technical specifications about the W&B Python Library, CLI, and Weave operations.
+## Start tracking models
+- **[Intro Notebook](http://wandb.me/intro)**: Track your first experiment with W&B with sample code
+- **[Integrations](./integrations/intro.md)**: Get started quickly with our integrations, which simplify tracking with popular libraries and frameworks, such as [PyTorch Lightning](./integrations/lightning.md), [Hugging Face](./integrations/huggingface.md), [XGBoost](./integrations/xgboost.md), and more.
 
-## How does W&B work?
 
-We recommend you read the following sections in this order if you are a first-time user of W&B:
+## W&B Products
```
Comment (ngrayluna) [inline docs/guides/intro.md]: Can we name this to "Explore the W&B Platform" or "Tools", or something similar? It makes the tone sound less market-y and changes it to an active tone (we like active tone in the docs) :).

File: docs/guides/intro.md [hunk 2]
```diff
@@ -1,36 +1,39 @@
...
+<iframe width="100%" height="330" src="https://www.youtube.com/embed/tHAFujRhZLA" title="Weights &amp; Biases End-to-End Demo" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
-If this is your first time using W&B we suggest you explore the following:
 
-1. Experience W&B in action, [run an example introduction project with Google Colab](http://wandb.me/intro).
-1. Read through the [Quickstart](../quickstart.md) for a quick overview of how and where to add W&B to your code.
-1. Read [How does W&B work?](#how-does-weights--biases-work) This section provides an overview of the building blocks of W&B.
-1. Explore our [Integrations guide](./integrations/intro.md) and our [W&B Easy Integration YouTube](https://www.youtube.com/playlist?list=PLD80i8An1OEGDADxOBaH71ZwieZ9nmPGC) playlist for information on how to integrate W&B with your preferred machine learning framework.
-1. View the [API Reference guide](../ref/README.md) for technical specifications about the W&B Python Library, CLI, and Weave operations.
+## Start tracking models
```
Comment (ngrayluna) [inline docs/guides/intro.md:17]: This title "seems" a little out of place...why don't we have a general call to action? We can call it "Next steps", "How to get started", etc.

**Comment by ngrayluna** [comment]
Overall looks really good. Main suggestions are:
1. Create a call to action
2.  Move the current section called "Start tracking models" -> "Next steps".


So the section order is
* Weights & Biases
   * Explore the W&B Platform
   * Next steps/How to get started/(whatever new title) <- Formally called "Start tracking models".
   * Resources

File: docs/guides/intro.md [hunk 3]
```diff
@@ -1,36 +1,39 @@
...
 
-1. Learn about [Runs](./runs/intro.md), W&B's basic unit of computation.
-2. Create and track machine learning experiments with [Experiments](./track/intro.md).
-3. Discover W&B's flexible and lightweight building block for dataset and model versioning with [Artifacts](./artifacts/intro.md).
-4. Automate hyperparameter search and explore the space of possible models with [Sweeps](./sweeps/intro.md).
-5. Manage the model lifecycle from training to production with [Model Management](./models/intro.md).
-6. Visualize predictions across model versions with our [Data Visualization](./data-vis/intro.md) guide.
-7. Organize W&B Runs, embed and automate visualizations, describe your findings, and share updates with collaborators with [Reports](./reports/intro.md).
+
+![](@site/static/images/general/wandb_diagram_july23.png)
```
Comment (ngrayluna) [inline docs/guides/intro.md:23]: Let's add a quick one liner explaining the diagram.

**Comment by ngrayluna** [comment]
In addition to the above, we'll want to re-word this to highlight the new product packaging. (And update w/ the new iamges).

**Comment by ngrayluna** [comment]
@cvphelps With this PR merged in https://github.com/wandb/docodile/pull/441  can we close this one?

**Comment by cvphelps** [comment]
Yes, closing this one

**Pull request closed without merging**